lib/flutter_login.dart

```diff
@@ -238,7 +238,7 @@ class FlutterLogin extends StatefulWidget {
       this.passwordValidator,
       this.onSubmitAnimationCompleted,
       this.logoTag,
-      this.userType = LoginUserType.email,
+      this.userType = LoginUserType.name,
```

Comment:
Why would we want to change this? I think it's most common to login with email instead of name.

Reply:
Just an option, some app might want using username instead.

Reply:
I understand, but I don't think we should change this as default behavior. If people want to use .name instead of .email they can just change the userType in their own implementation.